core-domain/src/main/java/gohigher/application/Application.java

```diff
@@ -22,4 +25,10 @@ public Application(String companyName, String location, String contact, String d
 			requiredCapability, preferredQualification, deadLine, processes, url);
 		this.currentProcess = currentProcess;
 	}
+
+	public static Application simple(String companyName, String duty, ProcessType processType, LocalDateTime schedule) {
+		Process process = new Process(processType, null, schedule);
+		return new Application(companyName, null, null, duty, null, null, null, null, null, null, null,
+			List.of(process), null, process);
+	}
```

Comment:
```suggestion
	public static Application simple(
		String companyName, 
		String duty, 
		ProcessType processType, 
		LocalDateTime schedule
	) {
		Process process = new Process(processType, null, schedule);
		return new Application(
			companyName, null, null, duty, null, null, 
			null, null, null, null, null,
			List.of(process), null, process
		);
	}
```

Application 특성상 데이터가 많아 어쩔 수 없긴 하지만.. 간단하게 개행이라도 추가하여 가독성을 높여보면 어떨까요? 예시로 추가하긴 했지만 병주님 편하신대로 반영해주세요.